docs/circuits.ipynb

```diff
@@ -579,6 +579,137 @@
     "Especially useful is dropping the last moment (which are often just measurements): `circuit[:-1]`, or reversing a circuit: `circuit[::-1]`.\n"
```

Comment:
<p>Does <code>CircuitOperation</code> add a link to the reference page or we need to write <code>cirq.CircuitOperation</code> ? Please change, here and elsewhere,  if it's the latter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5399/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
Done for the first appearance of `CircuitOperation` - I think this is our strategy in general, similar to Wikipedia-style linking.